docs/ops/sort/TopK_4.md

```diff
@@ -0,0 +1,116 @@
+`# TopK  {#openvino_docs_ops_sort_TopK_4}
+
+**Versioned name**: *TopK-4*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *TopK* computes indices and values of the *k* maximum/minimum values for each slice along a specified axis.
+
+**Attributes**
+
+* *axis*
+
+  * **Description**: Specifies the axis along which the values are retrieved.
+  * **Range of values**: An integer. Negative values means counting dimension from the back.
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *mode*
+
+  * **Description**: Specifies whether TopK selects the largest or the smallest elements from each slice.
+  * **Range of values**: "min", "max"
+  * **Type**: `string`
+  * **Required**: *yes*
+
+* *sort*
+
+  * **Description**: Specifies the order of corresponding elements of the output tensor.
+  * **Range of values**: `value`, `index`, `none`
+  * **Type**: `string`
+  * **Required**: *yes*
+
+* *stable*
+
+  * **Description**: Specifies whether the equivalent elements should maintain their relative order from the input tensor. Takes effect only if the `sort` attribute is set to `value` or `index`.
```

Comment:
Not sure how it will work for `index` case. Please check. It makes sense to me only in case `value` mode

Reply:
@rkazants Hi Roman! If I understand it correctly, for sorting by index, sometimes we don't have the chance to collect all the equivalent values in the stage of sorting by data, to do the next stage of sorting by index, because of the capacity of specific k. For such case, if we just pick one of these equivalent values randomly, that would violate our description "the equivalent elements should maintain their relative order from the input tensor". So I think the attribute of stable also make sense to sorting by index.

Here is the case I met, when I was implementing TopK-3 for CPU plugin.

```
Input tensor: {5, 3, 1, 2, 5, 5}, sort_mode=min, sort_type=indices, k=4

Possible result1:
Output data tensor: {5, 3, 1, 2}
Output index tensor: {0, 1, 2, 3}

Possible result2:
Output data tensor: {3, 1, 2, 5}
Output index tensor: {1, 2, 3, 4}

Possible result3:
Output data tensor: {3, 1, 2, 5}
Output index tensor: {1, 2, 3, 5}
```

If the attribute of stable is true, "Possible result1" is the only right result.

cc @tomdol @dmitry-gorokhov

Reply:
@xuchen-intel, you seem to be correct. How does the current version work for `sort_type=indices`? It does not guarantee "Possible result1", right?

I think we need to cover this case as well for TensorFlow TopK operations.
@xuchen-intel, how hard is it to add into the CPU plugin?

cc, @tomdol

Reply:
@rkazants Hi Roman! Current topk-11 in CPU plugin has already supported stable sorting for sort_type=indices. When stable is set for sort_type=indices, it guarantees "Possible result1", because sorting by indices reuses routines of sorting by values in CPU plugin.

cc @tomdol  @dmitry-gorokhov

Reply:
@xuchen-intel Hi Chen! That is just awesome that the CPU plugin already supports it.

@tomdol, can you please enable it on the core part and cover this case in the specification? I thin we should have it in this release to be aligned with TF.

Best regards,
Roman